apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/GrpcRdaSource.java

```diff
@@ -159,14 +159,40 @@ public void close() throws Exception {
     }
   }
 
+  public Metrics getMetrics() {
+    return metrics;
+  }
+
+  /**
+   * Indicates service is running but not actively processing a new record. Called at start of job
+   * and when a batch has been written.
+   */
+  @VisibleForTesting
+  void setUptimeToRunning() {
+    metrics.uptimeValue.set(10);
```

Comment:
should the values be constants?

Reply:
Could(maybe should) have done that.  In this case though the numbers don't really matter so much.  We just want to get a shape on a newrelic graph.  If we expand to more states I'd definitely want to make it a little more formalized.